specificatie/opschortingbijhouding.yaml

```diff
@@ -0,0 +1,22 @@
+openapi: 3.0.3
+info:
+  title: OpschortingBijhouding definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    OpschortingBijhoudingBasis:
+      type: object
+      properties:
+        reden:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+    OpschortingBijhouding:
```

Comment:
Is er een reden om deze uit elkaar te trekken ? (Reden en datum)

Reply:
basis wordt gebruikt in PersoonBeperkt en de normale in Persoon. Verder zie ik dat ook een GBA variant moet worden gemaakt

Reply:
GBA variant toegevoegd

Reply:
@MelvLee is er hier voldoende toegevoegde waarde om op OpschortingBijhouding polymorfisme toe te passen, alleen om in de specificaties te kunnen zeggen dat in de GBA variant alleen een GBA datum kan komen? Hier heeft de gebruiker (die achter de proxy zit) denk ik weinig aan, want in de versie die deze krijgt kan de datum alsnog een GBA datum zijn

Reply:
Als er geen informatie properties bijkomen is het niet nodig